engine/classes/ElggSession.php

```diff
@@ -244,6 +246,90 @@ public function remove($name) {
 	public function has($name) {
 		return $this->storage->has($name);
 	}
+	
+	/**
+	 * Logs in a specified \ElggUser. For standard registration, use in conjunction
```

Comment:
Please have a look at this description, I find it confusing

Reply:
me 2